chall04/vkuikka.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+import sys
+
+def main():
+	size = int(input())
+	world = []
+
+	if size == 0:
+		sys.exit()
+	for i in range(size):
+		world.append(input())
+		if len(world[i]) != size:
+			print("Wrong length")
+			sys.exit()
```

Comment:
you can pass the print into sys.exit argument ; https://docs.python.org/2/library/sys.html#sys.exit